### DISCUSSION TITLE
license lost

### DISCUSSION THREAD

**Discussion by Naty559** [aclap-dev/video-downloadhelper#1256] [License & Purchase requests]
### Your question

Good day, I bought a premium version of VDH Chrome in the Google Store before February 2021, and I did not receive a license key because it was linked to my Google account natyenriquez2503@gmail.com

I am attaching a payment email with proof of payment, please send me a new license key. 

Thank you

### Addon & CoApp version and details

Good day, I bought a premium version of VDH Chrome in the Google Store before February 2021, and I did not receive a license key because it was linked to my Google account natyenriquez2503@gmail.com

I am attaching a payment email with proof of payment, please send me a new license key. 

Thank you

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I am just another user.

You say you attached proof of purchase but there is no such attachment here on GitHub.  But that doesn't matter.  You can solve your own problem following instructions that are here on GitHub.  Start by clicking the link to the Wiki.  That link appears at the top of every page of this forum.  That is a sort of user manual for VDH.  You should get in the habit of referring to it regularly.  Once you are there, look for a table of contents down the right side of the page.  Each chapter is a link that you can click.  Over time, you should click every link & read everything you find.  For the moment, click the chapter with the title `Old Chrome licenses`.  That takes you to a page you should read.  The answer to your problem is on that page.

**Comment by jcfrog** [comment]
@Naty559 Hi , I found your purchase. We will send you a new license key.

**Reply by Naty559** [reply]
Hello, how are you, I still haven't received the new key, could you help me please?

**Reply by jcfrog** [reply]
I just replied to the email you sent me.
